tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
Nice, was just reading on this. So this works because we are on React 17, and this gives a bit of perf boost right?

Reply:
so i did this to try and address a build process bug that is associated with installing `@vercel/analytics`

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/12162433/198140251-e140709e-7588-4400-a271-ec2147899e7f.png">

unfortunately, vercel decided to revert this setting on my behalf! 😠 

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/12162433/198140385-dbdca72a-a44e-49dc-acdf-8058665f6c09.png">

supposedly an easy fix is to bump to React 18: https://github.com/hashicorp/next-mdx-remote/pull/250#issuecomment-1082395589